packages/botonic-core/src/handoff.js

```diff
@@ -151,3 +161,12 @@ export async function getAvailableAgents(session) {
   })
   return resp.data
 }
+
+export function cancelHandoff(session, typification = null) {
```

Comment:
once Caroline mentioned that [typification](https://en.wikipedia.org/wiki/Typification) was not a correct word for this. I suggest [classification](https://www.collinsdictionary.com/es/diccionario/espanol-ingles/tipificaci%C3%B3n)

Reply:
I agree, but I prefer to do this refactor in botonic and hubtype at the same time. Leaving it like this for now.